content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml

```diff
@@ -115,6 +115,42 @@
                             </column>
                         </items>
                     </socialmedia>
+                    <datalayer
+                        cq:showOnCreate="{Boolean}true"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Data Layer"
+                        sling:orderBefore="cloudservices"
+                        sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
+                        <items jcr:primaryType="nt:unstructured">
+                            <column
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/container">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <section
+                                        jcr:primaryType="nt:unstructured"
+                                        jcr:title="Data Layer Settings"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <primary
+                                                cq:showOnCreate="{Boolean}true"
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Primary type of page e.g. Products"
+                                                fieldLabel="Page primary category"
+                                                name="./primaryCategory"/>
+                                            <secondary
+                                                cq:showOnCreate="{Boolean}true"
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Secondary type of page e.g. Surfing"
```

Comment:
Having examples seems odd here. If im an FSI customer, it seems odd having an example that has nothing to do w/ my business. Perhaps, these should be more descriptive; What do "Primary and Secondary" types *mean* .. whats the intent of capturing them.